.github/workflows/nodejs.yml

```diff
@@ -14,12 +14,12 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v4
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
```

Comment:
This imo is like pinning deps in package.json; not worth the trouble. We won’t get bugfixes and whatnot automatically by doing this.

Reply:
Unless this bot also sends PRs (or some other bot does)

Reply:
dependabot works with pinned actions too. We're extensively using it already on nodejs/core. See: https://github.com/nodejs/security-wg/pull/1126

Reply:
It does indeed, but that makes it very noisy. I don’t think it’s worth it, especially from official Github actions.